opus/import/obs_surface_geometry_target.py

```diff
@@ -32,37 +32,46 @@ def field_obs_surface_geometry_target_iau_east_longitude1(self):
         long = self.field_obs_surface_geometry_target_iau_west_longitude2()
         if long is None:
             return None
-        return 360. - long
+        return (360. - long) % 360
 
     def field_obs_surface_geometry_target_iau_east_longitude2(self):
         long = self.field_obs_surface_geometry_target_iau_west_longitude1()
         if long is None:
             return None
-        return 360. - long
+        if (long == 0 and
+            self.field_obs_surface_geometry_target_iau_west_longitude2() == 360):
+            return 360
+        return (360. - long) % 360
 
     def field_obs_surface_geometry_target_sub_solar_iau_east_longitude1(self):
         long = self.field_obs_surface_geometry_target_sub_solar_iau_west_longitude2()
         if long is None:
             return None
-        return 360. - long
+        return (360. - long) % 360
 
     def field_obs_surface_geometry_target_sub_solar_iau_east_longitude2(self):
         long = self.field_obs_surface_geometry_target_sub_solar_iau_west_longitude1()
         if long is None:
             return None
-        return 360. - long
+        if (long == 0 and
+            self.field_obs_surface_geometry_target_sub_solar_iau_west_longitude2() == 360):
+            return 360
+        return (360. - long) % 360
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/SETI/rms-opus/pull/1378#issuecomment-2276940859>#1c82e7</a><span> - 08/09/2024, 12:57 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The condition 'if (long == 0 and self.field_obs_surface_geometry_target_sub_solar_iau_west_longitude2() == 360):' is unnecessarily complex due to the parentheses around the 'and' condition. This can lead to confusion and potential errors in understanding the code.
</div>
<div id="fix">
<b>Fix:</b> Simplify the conditional check by removing unnecessary parentheses around the 'and' condition.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -56,4 +56,4 @@
       if long is None:
           return None
 -     if (long == 0 and
 +     if long == 0 and
           self.field_obs_surface_geometry_target_sub_solar_iau_west_longitude2() == 360:
           return 360
       return (360. - long) % 360
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged